vue/components/ui/atoms/dropdown/dropdown.vue

```diff
@@ -143,11 +143,17 @@
     width: auto;
 }
 
-.dropdown-container .dropdown > .dropdown-item:hover,
-.dropdown-container .dropdown > .dropdown-item:active,
+.dropdown-container .dropdown > .dropdown-item:not(.disabled):hover,
+.dropdown-container .dropdown > .dropdown-item:not(.disabled):active,
 .dropdown-container .dropdown > .dropdown-item.selected,
-.dropdown-container .dropdown > .dropdown-item.highlighted {
+.dropdown-container .dropdown > .dropdown-item:not(.disabled).highlighted {
     background-color: $soft-blue;
+    font-weight: 700;
+}
+
+.dropdown-container .dropdown > .dropdown-item.disabled {
+    cursor: not-allowed;
+    opacity: 0.5;

```

Comment:
in the frontend we sometimes also add `pointer-events: none;` css line to enforce the a action on click event which in this case might make sense too
```suggestion
    opacity: 0.5;
    pointer-events: none;
```

Reply:
nevermind, in this case the cursor would not show as we'd want